package.json

```diff
@@ -8,8 +8,10 @@
     "coverage": "nyc report --reporter=text",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint \"**/*.js\" --ignore-pattern node_modules/ --fix",
-    "build": "babel ./src -d dist",
-    "start": "node dist/index.js",
+    "clean": "rm -rf ./dist && mkdir dist",
+    "build": "npm run clean && babel ./src -d dist",
+    "heroku-postbuild": "sequelize db:migrate:undo && sequelize db:migrate && sequelize db:seed:all",
```

Comment:
Why `run` or `undo` migrations and why `seed` after Heroku builds? This means the DB will be cleaned up after each build, then what happens to our DB data? Lost. please remove.

Reply:
to run migration use the `migrate:prod` script because there the `NODE_ENV`= `production`.

Reply:
@OlawaleJoseph @Dolapo47 @zanio @conquext

Reply:
We are currently undoing the migrations and seeding because we are still building and we don't want multiplication of data in the database whenever we push to heroku. Without undoing we will keep seeding the same data to the dB over and over again

Reply:
> Why `undo` migrations and why `seed` after Heroku builds? This means the DB will be cleaned up after each build, then what happens to our DB data? Lost. please remove.

Presently we are seeding the database to enable testing as some functionalities have not been implemented yet and also data such as roles will be seeded.

Reply:
> We are currently undoing the migrations and seeding because we are still building and we don't want multiplication of data in the database whenever we push to heroku. Without undoing we will keep seeding the same data to the dB over and over again

that's not the best way to go about it. you should run the `undo migration` and `seed db` scripts manually on Heroku. If we allow this, then there is a high chance of this script slipping through to `prod` and we do not want that.

Reply:
Alright. Will implement this ASAP

Reply:
Changes implemented